test/healthcheck_test.go

```diff
@@ -261,12 +306,12 @@ func (s) TestHealthCheckHealthServerNotRegistered(t *testing.T) {
 	cc, r := setupClient(t, nil)
 	r.UpdateState(resolver.State{
 		Addresses: []resolver.Address{{Addr: lis.Addr().String()}},
-		ServiceConfig: parseServiceConfig(t, r, `{
+		ServiceConfig: parseServiceConfig(t, r, fmt.Sprintf(`{
 	"healthCheckConfig": {
 		"serviceName": "foo"
 	},
-	"loadBalancingConfig": [{"round_robin":{}}]
-}`)})
+	"loadBalancingConfig": [{"%s":{}}]
```

Comment:
We didn't do a good job of indenting these JSON strings correctly in old test code. While you are here, could we please have them indented properly so that the arguments to things like `fmt.Sprintf` show up in a more readable way.

Reply:
Done.